backend/sql/initial_schema.sql

```diff
@@ -0,0 +1,100 @@
+create table project
+(
+    project_id  bigserial     not null primary key,
+    name        varchar(256)  not null unique,
+    description varchar(1000) null
+);
+
+create table test
+(
+    test_id bigserial    not null primary key,
+    name    varchar(256) not null unique
+);
+
+create table subtest
+(
+    subtest_id      bigserial      not null primary key,
+    test_id         bigint         not null,
+    name            varchar(256)   not null unique,
+    -- Method
+    meth            varchar(10000) null,
+    confluence_link varchar(1000)  null,
+    constraint belongs_to_test
+        foreign key (test_id) references test (test_id)
+            on update cascade
```

Comment:
Do we need `on update cascade` here? AFAIU `test_id` is immutable, so if it's somehow updated, we'd like to get an error.

Regarding `on delete cascade`, AFAIU it means that someone may delete a test and automatically delete all its subtests. Maybe that's what we want, maybe we want to explicitly remove all subtests first, I don't know. Probably that's too minor for now, let's consider it later.

Reply:
we can make it `no action` (almost the same as `restrict`)